### PR TITLE
increasing arithmetic input width from style

### PR DESCRIPTION
**Pull request by ShilpaSivanesan**

### CODE REVIEW

**Comment by ShilpaSivanesan** [comment]
@Gauravshah as grafana doesn't allow class more than width-30, added style parameter and increased the width to make it look uniform

File: src/partials/query.editor.html [hunk 1]
```diff
@@ -57,7 +57,7 @@
       <label class="gf-form-label query-keyword width-10">
         Expression
       </label>
-      <input type="text" class="gf-form-input width-20" ng-model="ctrl.target.expression" spellcheck='false' placeholder="query" ng-blur="ctrl.targetBlur()"  />
+      <input type="text" class="gf-form-input" style="width-1200px" ng-model="ctrl.target.expression" spellcheck='false' placeholder="query" ng-blur="ctrl.targetBlur()"  />
```
Comment (Gauravshah) [inline src/partials/query.editor.html]: this would not look good on screen sizes

**Comment by Gauravshah** [comment]
i am not sure how to deal with max-30

File: src/partials/query.editor.html (hunk 1 above)
Comment (ShilpaSivanesan) [reply src/partials/query.editor.html]: shall i just use grafana max width of 30?

**Pull request closed without merging**